src/components/blogContent/BlogContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { makeStyles, TextField, Typography } from "@material-ui/core";
+import React from "react";
+const useStyles = makeStyles((theme) => ({
+  "padding-md": {
+    padding: theme.spacing(10),
+  },
+  "padding-sm": {
+    padding: theme.spacing(3),
+  },
+}));
+const BlogContet: React.FC = () => {
+  const classes = useStyles();
+  const contents = ["Title", "Content", "Blurb/Summary", "Tags"];
+  return (
+    <div className={classes["padding-md"]}>
+      {contents.map((content, idx) => (
+        <div className={classes["padding-sm"]} key={`${content}-${idx}`}>
+          <Typography variant={"h3"}>{`${content}:`}</Typography>
+          <TextField
+            fullWidth
+            label={`${content}:`}
+            id={`${content}Id-${idx}`}
+            variant="outlined"
+            multiline={content === "Content"}
+            minRows={content === "Content" ? 10 : 0}
```

Comment:
I like the idea you're having here of reducing the amount of replication, but I think sometimes there are benefits to have a bit of duplication when it highlights the construction of the components and prevents us from having to have edge-case logic (like the `content === "Content"` bit). Why don't we go ahead and instead either create a component which holds all of the div + typography + text field (and allows us to pass in minRows and multiLine flags) or we can explicitly write this out for the 4 sections.

Reply:
Updated